
Copyright notice for examples/graph/cholesky/CMakeLists.txt:

# Copyright (c) 2020-2021 Intel Corporation
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

diff -urp oneTBB-2021.2.0.orig/examples/graph/cholesky/CMakeLists.txt oneTBB-2021.2.0/examples/graph/cholesky/CMakeLists.txt
--- oneTBB-2021.2.0.orig/examples/graph/cholesky/CMakeLists.txt	2021-06-28 00:12:40.615448769 -0700
+++ oneTBB-2021.2.0/examples/graph/cholesky/CMakeLists.txt	2021-06-28 00:20:56.654634051 -0700
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# oteodoro - fix missing mkl_cblas.h on Gentoo Linux systems
+
 cmake_minimum_required(VERSION 3.1)
 
 project(cholesky CXX)
@@ -21,6 +23,7 @@ include(../../common/cmake/common.cmake)
 set_common_project_settings(tbb)
 
 # TODO: Consider using FindMKL module
+include_directories(/usr/include/mkl)							# oteodoro - added line
 find_library(MKL_INTEL_LP64_LIB mkl_intel_lp64 PATHS ENV LIBRARY_PATH)
 find_library(MKL_SEQUENTIAL_LIB mkl_sequential PATHS ENV LIBRARY_PATH)
 find_library(MKL_CORE_LIB       mkl_core PATHS ENV LIBRARY_PATH)
